ts/utils/hooks/bottomSheet.tsx

```diff
@@ -162,6 +168,12 @@ export const useIOBottomSheetModal = ({
   );
 
   useEffect(() => {
+    // Check if the screen reader is enabled when the component is mounted
+    isScreenReaderEnabled()
+      .then(setIsScreenReaderEnabled)
+      .catch(_ => setIsScreenReaderEnabled(false));
+    // Subscribe to `screenReaderChanged` to properly update the state.
+    // The method above is necessary because of this event execute the given function only when the screen reader internal state changes; unfortunately its function is not executed on subscription.
```

Comment:
```suggestion
    // The method above is necessary because the event listener is triggered 
    // only when the screen reader internal state changes.
    // Unfortunately its function is not executed on subscription.
```